src/ctapipe_io_nectarcam/__init__.py

```diff
@@ -811,18 +908,21 @@ def fill_trigger_info(self, array_event):
                 f'Event {array_event.index.event_id} has unknown event type, trigger: {trigger_bits:08b}')
             trigger.event_type = EventType.UNKNOWN
 
-    def unpack_feb_data(self, event_container, event):
+    def unpack_feb_data(self, event_container, event, nectarcam_data):
         '''Unpack FEB counters and trigger pattern'''
 
         # Deduce data format version
         bytes_per_module = len(
-            event.nectarcam.counters) // self.camera_config.nectarcam.num_modules
+            nectarcam_data.counters) // self.nectarcam_service.num_modules
+        # VIM NOTES> There is in v6 the num_modules per event... Use this one ? 
```

Comment:
I'm not sure, is this discussed in the documentation? Could this change if there are missing modules?